tests/e2e/support/commands.js

```diff
@@ -197,6 +247,18 @@ Cypress.Commands.add('leaveLobbyOpponent', (id) => {
     });
   });
 });
+
+Cypress.Commands.add('recoverSessionOpponent', (username) => {
+  return new Cypress.Promise((resolve, reject) => {
+    io.socket.get('/test/recoverSession', { username }, function handleResponse(res, jwres) {
+      if (jwres.statusCode !== 200 || res === false) {
+        return reject(new Error('error recovering session'));
+      }
+      return resolve();
+    });
+  });
+});
```

Comment:
This might work by hitting the existing reLogin endpoint, which would save the need to add a new testing-only endpoint. I'd also like to revise the command itself to input a player object with username and password so it can accept the user fixtures exactly as exported in the fixture file